cmake/templates/pkgConfig.cmake.in

```diff
@@ -76,6 +76,10 @@ else()
   set(@PROJECT_NAME@_DEVEL_PREFIX "")
   set(@PROJECT_NAME@_INSTALL_PREFIX @CMAKE_INSTALL_PREFIX@)
   set(@PROJECT_NAME@_PREFIX ${@PROJECT_NAME@_INSTALL_PREFIX})
+  set(@PROJECT_NAME@_INSTALL_BINDIR @CMAKE_INSTALL_BINDIR@)
+  set(@PROJECT_NAME@_INSTALL_LIBDIR @CMAKE_INSTALL_LIBDIR@)
+  set(@PROJECT_NAME@_INSTALL_DATADIR @CMAKE_INSTALL_DATADIR@)
```

Comment:
Could also be `DATAROOTDIR`, but nixpkgs seems to prefer `DATADIR`, so I went with that. They have the same value.